lib/api/tenure/v1/types.ts

```diff
@@ -65,36 +70,49 @@ export interface LegacyReference {
   name: string;
   value: string;
 }
-
+export interface FurtherAccountInformation {
+  isRentAccountRequired: boolean | null;
+  noRentAccountReason: string | null;
```

Comment:
String are not required to be nullable so please remove that

Reply:
@humulla I might be missing something here but if we're compiling in strict would we not still need to set null?

Reply:
I misread this part having nullable should be fine